site/docs/v1/tech/events-webhooks/index.adoc

```diff
@@ -22,23 +22,24 @@ See the corresponding link:/docs/v1/tech/apis/webhooks[Webhook APIs] if you'd pr
 
 Here are the topics in this section:
 
-* link:/docs/v1/tech/events-webhooks/events/[Events] - Covers all of the event types that FusionAuth sends to Webhooks
 * link:/docs/v1/tech/events-webhooks/writing-a-webhook[Writing a Webhook] - Covers how to write a Webhook that will process events from FusionAuth.
 * link:/docs/v1/tech/events-webhooks/securing[Securing a Webhook] - Covers how to ensure your webhooks are secured properly.
+* link:/docs/v1/tech/events-webhooks/events/[Events] - Covers all of the event types that FusionAuth sends to Webhooks
```

Comment:
Assume you moved this down to bring it in line with the nav?

Reply:
Yeah.